### PR TITLE
Edit KubeMacPoolDuplicateMacsFound

### PR DESCRIPTION
**Pull request by apinnick**
Signed-off-by: Avital Pinnick <apinnick@redhat.com>

### CODE REVIEW

**Comment by apinnick** [comment]
@sradco @nunnatsa 
Can someone review this PR?

**Comment by sradco** [comment]
/lgtm
@RamLavi Can you please review?

File: docs/runbooks/KubeMacPoolDuplicateMacsFound.md [hunk 1]
```diff
@@ -1,32 +1,40 @@
...
-Upon starting KubeMacPool, it scans the cluster for MAC Addresses in VMs set on the managed namespaces.  
-In case it finds duplicates it will raise this alert.
+This alert fires when `KubeMacPool` detects duplicate MAC addresses.
+
+`KubeMacPool` is responsible for allocating MAC addresses and preventing MAC address conflicts. When `KubeMacPool` starts, it scans the cluster for the MAC addresses of virtual machines (VMs) in managed namespaces.
 
 ## Impact
 
-Duplicate MAC Addresses on the same LAN may cause unexpected and various connectability issues in the network.
+Duplicate MAC addresses on the same VLAN might cause network issues.
```
Comment (RamLavi) [inline docs/runbooks/KubeMacPoolDuplicateMacsFound.md:12]: While in theory a network can have duplicate MAC addresses if the VLAN on the NADs is different - KMP does not make that distinction. There is no relevance to VLAN here.
```suggestion
Duplicate MAC addresses on the same LAN might cause network issues.
```

File: docs/runbooks/KubeMacPoolDuplicateMacsFound.md [hunk 2]
```diff
@@ -1,32 +1,40 @@
...
-    - The Mac Address which has a conflict.
-    - The current entry in the data base (VM namespace, name and interface).
-    - The entry that rejected due to conflict (VM namespace, name and interface).  
+1. Obtain the namespace and the name of the `kubemacpool-mac-controller` pod:
+  ```bash
+  $ kubectl get pod -A -l control-plane=mac-controller-manager --no-headers \
+    -o custom-columns=":metadata.namespace,:metadata.name"
+  ```
+
+1. Obtain the duplicate MAC addresses from the `kubemacpool-mac-controller` logs:
```
Comment (RamLavi) [inline docs/runbooks/KubeMacPoolDuplicateMacsFound.md:22]:
```suggestion
2. Obtain the duplicate MAC addresses from the `kubemacpool-mac-controller` logs:
```

**Review by RamLavi — CHANGES_REQUESTED** [review]

File: docs/runbooks/KubeMacPoolDuplicateMacsFound.md (hunk 1 above)
Comment (apinnick) [reply docs/runbooks/KubeMacPoolDuplicateMacsFound.md:12]: Fixed

File: docs/runbooks/KubeMacPoolDuplicateMacsFound.md (hunk 2 above)
Comment (apinnick) [reply docs/runbooks/KubeMacPoolDuplicateMacsFound.md:22]: Numbering for markdown can be "1." for each step. It will resolve correctly.

Comment (RamLavi) [reply docs/runbooks/KubeMacPoolDuplicateMacsFound.md:21]: I don't follow..
you want it to be lie this?
```
1. Obtain the namespace and the name of the kubemacpool-mac-controller pod:
...
1. Obtain the duplicate MAC addresses from the kubemacpool-mac-controller logs:
```

if not - please fix. If you do - then why? these are consecutive items in the same list, why would they have the same number?

Comment (apinnick) [reply docs/runbooks/KubeMacPoolDuplicateMacsFound.md:21]: Normally markdown does not require different number for ordered lists.
image

If you merge this and the numbers are not correct, I will fix them.

**Comment by RamLavi** [comment]
@apinnick @sradco note post merge the numbering is not fixed:
https://github.com/kubevirt/monitoring/blob/main/docs/runbooks/KubeMacPoolDuplicateMacsFound.md

**Pull request merged**